src/_asset/font/font.scss

```diff
@@ -26,6 +26,11 @@
   -moz-osx-font-smoothing: grayscale;
 }
 
+.icon__bell {
+  &:before {
+    content: $icon__bell; 
+  }
+}
```

Comment:
Empty lines before and after for readability. 
Ps. Pls don't be mad at me 😅😌

Reply:
It's WIP and it's been saved as a draft 😃 🤗

Reply:
I know, i saw that.